### PR TITLE
fix paper claims and log in billing manager, add fax for insurance companies in practice settings

### PR DESCRIPTION
**Pull request by stephenwaite**
Fixes #
wasn't writing log output to process_bills.log in billing manager

#### Short description of what this resolves:
now writing

### CODE REVIEW

File: interface/billing/billing_process.php [hunk 1]
```diff
@@ -60,6 +60,13 @@
...
 <?php
-    $logger->onLogComplete();
+$logger->onLogComplete();
+$hlog = $logger->hlog();
+if (!empty($hlog)) {
+    if ($GLOBALS['drive_encryption']) {
+        $hlog = $cryptoGen->encryptStandard($hlog, null, 'database');
+    }
+    file_put_contents($GLOBALS['OE_SITE_DIR'] . "/documents/edi/process_bills.log", $hlog);
+}
```
Comment (bradymiller) [inline interface/billing/billing_process.php:70]: yep, being able to actually write to the log is a useful thing to support :)

**Review by bradymiller — APPROVED** [review]

File: interface/billing/billing_process.php (hunk 1 above)
Comment (kchapple) [reply interface/billing/billing_process.php:70]: I would probably put this in the BillingProcessor class but we can always clean up later.

**Comment by kchapple** [comment]
Thanks @stephenwaite ! You don’t need to pass the hlog parameter to the logger though, because it’s in the Logger class. You can just use the hlog property that’s in the class like $this->hlog in onLogComplete() rather than passing it in in billing_process.php. Thanks for implementing this!

**Comment by stephenwaite** [comment]
PICKLE

**Review by bradymiller — APPROVED** [review]

**Comment by stephenwaite** [comment]
thank you @bradymiller , here's some final fixes @kchapple

**Comment by stephenwaite** [comment]
testing well and bringing in

**Comment by bradymiller** [comment]
2DV (1)

**Pull request merged**